### ISSUE TITLE
ToSchema instance of NominalDiffTime doesn't match the description in Data.Time.Clock

### ISSUE THREAD

**Issue by fierce-katie** [GetShopTV/swagger2#99]
There's the following instance:
```
instance ToSchema NominalDiffTime where
  declareNamedSchema _ = declareNamedSchema (Proxy :: Proxy Integer)
```
Meanwhile [`time` documentation](http://hackage.haskell.org/package/time-1.4.0.1/docs/Data-Time-Clock.html#t:NominalDiffTime) says :

> This is a length of time, as measured by UTC. Conversion functions will treat it as seconds. It has a precision of 10^-12 s.

It leads to unexpected errors when I try to validate  `ToJSON` matches `ToSchema`:
```
>>> validateToJSON (10::NominalDiffTime)
[]
>>> validateToJSON (10.1::NominalDiffTime)
["not an integer"]
```

**Comment by fizruk** [comment]
Indeed. We need to add `ToSchema` and `ToParamSchema` instances for [`Fixed`](https://hackage.haskell.org/package/base-4.9.1.0/docs/Data-Fixed.html).
Then we can redefine `ToSchema` instance for `NominalDiffTime` in terms of `Pico` (which is a type synonym for `Fixed E12`).

**Comment by fierce-katie** [comment]
Isn't it fixed already (see #121)?

**Comment by fierce-katie** [comment]
Fixed in [`swagger2-2.1.6`](http://hackage.haskell.org/package/swagger2-2.1.6).
